tests/tasks/test_norefposterior.py

```diff
@@ -0,0 +1,402 @@
+import numpy as np
```

Comment:
Great! Can we factor some of these tests out into a separate PR that adds generic tests for all tasks?

Reply:
Can you make a concrete proposal? 
My suggestion: I am happy to send another PR for `test_two_moons.py` while keeping `test_norefposterior.py`. 
What do you think?

Reply:
I will hopefully have time to work on the tests and #23 tomorrow, or Wednesday latest, and make a concrete proposal then. Cheers

Reply:
Cheers for opening https://github.com/sbi-benchmark/sbibm/pull/29 -- I'd suggest we parametrize those tests to run for all tasks, that's what I had in mind when I asked about "generic tests" above

Reply:
ok, I propose to merge this first and then adopt it in #29 or a subsequent PR.